src/index.js

```diff
@@ -1 +1,78 @@
 import './index.css';
+
+// Game ID
+const gameId = 'Af3SPW0U28yG92UyBksL';
+
+// API URL
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
+const refreshScores = async () => {
+  try {
+    const response = await fetch(`${apiUrl}/games/${gameId}/scores`);
+    const data = await response.json();
+
+    // Update the score list
+    const scoreList = document.querySelector('.scorelist');
+    scoreList.innerHTML = '';
+
+    data.result.forEach((score) => {
+      const listItem = document.createElement('li');
+      listItem.textContent = `${score.user}:${score.score}`;
+      scoreList.appendChild(listItem);
+    });
+  } catch (error) {
+    console.error('Error refreshing scores:', error);
```

Comment:
- [ ] Please do not commit **console.log** to your repo, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. Few users keep the dev tools section open to monitor errors and therefore they mate not see the error intended for them.

**My suggestion are**
 - [ ] Delete the console.log statements
 - [ ] add a p tag to the document and write the error messages in that p tag using JavaScript. By doing like this, the error messages will be displayed where the user can see 👍🏾 **Happy coding**